apps/laboratory/src/pages/library/multichain-ethers-solana-siwe.tsx

```diff
@@ -3,20 +3,14 @@ import { EthersAdapter } from '@reown/appkit-adapter-ethers'
 import { SolanaAdapter } from '@reown/appkit-adapter-solana'
 import { ThemeStore } from '../../utils/StoreUtil'
 import { ConstantsUtil } from '../../utils/ConstantsUtil'
-import {
-  mainnet,
-  polygon,
-  solana,
-  arbitrum,
-  optimism,
-  solanaTestnet,
-  solanaDevnet
-} from '@reown/appkit/networks'
 import { AppKitButtons } from '../../components/AppKitButtons'
 import { HuobiWalletAdapter, SolflareWalletAdapter } from '@solana/wallet-adapter-wallets'
 import { MultiChainTestsEthersSolana } from '../../components/MultiChainTestsEthersSolana'
 import { siweConfig } from '../../utils/SiweUtils'
 import { SiweData } from '../../components/Siwe/SiweData'
+import { mainnet } from '@reown/appkit/networks'
+
+const networks = [...ConstantsUtil.EvmNetworks, ...ConstantsUtil.SolanaNetworks]
```

Comment:
And similar for this, like
```suggestion
const networks = ConstantsUtil.EvmSolanaNetworks
```

And we can vary when we have more chains, like

```js
const networks = ConstantsUtil.EvmPolkadotNetworks
```

But prob when we have more chains, we wouldn't have all variations of networks, I mean instead of:
- wagmi + solana
- wagmi + polkadot
- wagmi + bitcoin

We'd only have 
- wagmi + solana + polkadot + ..whatever we have

In this case `ConstantsUtil.AllNetworks` would work

Reply:
I think we should not over-abstract all these combinations. We can have a mixer function later when we have more variants. 

I think `AllNetworks` is valuable tho, addded it 🫡